packages/webapp/pages/account/profile.tsx

```diff
@@ -83,6 +88,165 @@ const AccountProfilePage = (): ReactElement => {
     },
   );
 
+  const RenderForm = () => (
+    <form ref={formRef} id="submit-profile">
+      <AccountContentSection
+        className={{ heading: 'mt-0' }}
+        title="Profile Picture"
+        description="Upload a picture to make your profile stand out and let people recognize
+        your comments and contributions easily!"
+      >
+        <div className="relative mt-6 flex">
+          <ImageInput
+            id={imageId}
+            className={{
+              img: 'object-cover',
+              container: 'border-4 !border-background-default',
+            }}
+            initialValue={user.image}
+            hoverIcon={<CameraIcon size={IconSize.Large} />}
+            onChange={(_, file) => {
+              if (!file) {
+                return;
+              }
+
+              updateUserProfile({
+                image: file,
+              });
+            }}
+          />
+          <ImageInput
+            id={coverId}
+            className={{
+              root: 'absolute left-0 top-0 flex w-full',
+              container:
+                'border-0 bg-background-subtle hover:bg-accent-pepper-subtlest',
+              img: 'object-cover',
+            }}
+            size="cover"
+            initialValue={currentCoverImage}
+            fallbackImage={null}
+            alwaysShowHover={!currentCoverImage}
+            hoverIcon={
+              <span className="ml-26 mr-3 flex flex-wrap items-center justify-center text-theme-label-secondary">
+                <CameraIcon size={IconSize.Large} />
+                <span className="ml-1.5 font-bold typo-callout">
+                  Upload cover image
+                </span>
+              </span>
+            }
+            fileSizeLimitMB={5}
+            onChange={(fileName, file) => {
+              setCoverImage(fileName);
+              if (!file) {
+                return;
+              }
+
+              uploadCoverImage({
+                image: file,
+              });
+            }}
```

Comment:
Same as above, since this is quite a bit component, maybe let's move this to a func so it can be read easily?